### DISCUSSION TITLE
Pin Multiplexing on Seeed Studio XIAO ESP32C3 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#153] [Q&A]
# Pin Multiplexing on Seeed Studio XIAO ESP32C3 | Seeed Studio Wiki

Pin Multiplexing on Seeed Studio XIAO ESP32C3

https://wiki.seeedstudio.com/XIAO_ESP32C3_Pin_Multiplexing/#comments

**Comment by Larry-Home** [comment]
Have been trying to make use of the "Special way - use USB serial and UART0/UART1 at the same time" section so that we can hook up to the Longan Serial CAN adapter.  I have literally tried everything and cannot seem to get a response on either of the UART devices.  Have seen some reports that there are problems with the ESP32C3 and UART ports, but not sure how to test that theory.  I am utilizing D4 /D5 as I originally thought I was getting the I2C version but it is not.  From what I can tell this should not be an issue.

**Comment by cramlr** [comment]
Aloha, I can only seem to get 5 PWM outputs at a time.  I need 6.  How many are supported simultaneously?

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
